### PR TITLE
[PAN-2994] Add responsible disclosure policy.

### PR DESCRIPTION
**Pull request by bgravenorst**
## PR description
Add Security Disclosure Policy to the Pantheon documentation.

## Fixed Issue(s)
PAN-2994

### CODE REVIEW

**Review by shemnon — APPROVED** [review]

File: docs/Security/Responsible-Disclosure.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+description: Pantheon responsible disclosure statement.
+<!--- END of page meta data -->
+
+# Responsible Disclosure Policy
+
+At PegaSys, we consider security to be a priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
At PegaSys, security is a priority. But regardless of how much effort we put into system security, there may still be vulnerabilities present.
```

File: docs/Security/Responsible-Disclosure.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+If you discover a vulnerability, we would like to know about it so we can take steps to address it as quickly as possible. We would like to ask you to help us better protect our clients and our systems.
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
If you discover a vulnerability, we want to know about it so we can take steps to address it as quickly as possible. We would like to ask you to help us better protect our clients and our systems.
```

File: docs/Security/Responsible-Disclosure.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+ 
+Please do the following:
+
+* E-mail your findings to security@pegasys.tech and provide sufficient information to reproduce the problem, so we can resolve it as quickly as possible.
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
* E-mail your findings to security@pegasys.tech. Provide sufficient information to reproduce the problem so we can resolve it as quickly as possible.
```

File: docs/Security/Responsible-Disclosure.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* Do not take advantage of the vulnerability or problem you have discovered.
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
* Do not take advantage of the vulnerability you have discovered.
```

File: docs/Security/Responsible-Disclosure.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* Practice responsible disclosure, in other words, don’t reveal the problem to others until either:
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
* Practice responsible disclosure. That is, don’t reveal the problem to others until either:
```

File: docs/Security/Responsible-Disclosure.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    * We have released a fix for the disclosure, or 
+    * 90 days have passed, or 
+    * We waive responsible disclosure. 
+ 
+We will acknowledge receipt of your vulnerability report the next business day and strive to send you regular updates about our progress.
```
Comment (MadelineMurray) [inline docs/Security/Responsible-Disclosure.md]:
```suggestion
We will acknowledge receipt of your vulnerability report the next business day and send you regular updates about our progress.
```

**Review by MadelineMurray — APPROVED** [review]
Edits to simplify language.

**Pull request merged**